src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -63,30 +63,45 @@ export const setEmptyCalendarValues = calendar => {
 
 export const mergeCalendarsContributions = (
   actualCalendar,
-  gitHubUserJsonCalendar
+  gitHubUserJsonCalendar,
+  gitHubUserTimeFrame
 ) => {
   const copiedActualCalendar = javaScriptUtils.deepCopyObject(actualCalendar);
 
   gitHubUserJsonCalendar.children[0].children.forEach(
     (weeklyData, weekIndex) => {
       weeklyData.children.forEach((dailyData, dayIndex) => {
-        if (dailyData.attributes["data-count"]) {
-          const actualCalendarDailyData = calendarUtils.getCalendarDataByIndexes(
-            copiedActualCalendar,
-            weekIndex,
-            dayIndex
-          );
-          const totalDailyContributions =
-            Number(actualCalendarDailyData.attributes["data-count"]) +
-            Number(dailyData.attributes["data-count"]);
-
-          copiedActualCalendar.children[0].children[weekIndex].children[
-            dayIndex
-          ].attributes = {
-            ...actualCalendarDailyData.attributes,
-            "data-count": String(totalDailyContributions),
-            fill: calendarUtils.getFillColor(totalDailyContributions)
-          };
+        const dayDate = new Date(dailyData.attributes["data-date"]);
+        const timeFrameDateFrom = gitHubUserTimeFrame
+          ? new Date(gitHubUserTimeFrame[0])
+          : dayDate;
+        const timeFrameDateTo = gitHubUserTimeFrame
+          ? new Date(gitHubUserTimeFrame[1])
+          : dayDate;
+
+        if (
+          dailyData.attributes.class === "day" &&
```

Comment:
AFAICT this conditional is redundant because of:

```js
if (dailyData.attributes["data-count"]) { ... }
```

I don't really remember off the top of my head but IIRC the above-mentioned conditional was required to disregard the months when merging the calendars into each other.

Reply:
> I don't really remember off the top of my head but IIRC the above-mentioned conditional was required to disregard the months when merging the calendars into each other.

Yep, that's it, but let's rather use `dailyData.attributes.class === "day"`.